string      rover_name
float64[]   data
